unlock-app/src/pages/_document.jsx

```diff
@@ -7,6 +7,12 @@ export default class MyDocument extends Document {
     return (
       <Html lang="en">
         <Head>
+          <script defer src="https://js.stripe.com/v3/" />
```

Comment:
Stripe requires these to be loaded...

Reply:
need eventually to consider also who uses unlock externally without our app